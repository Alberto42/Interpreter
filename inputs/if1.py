x = 0
if True {
    x = 1
}