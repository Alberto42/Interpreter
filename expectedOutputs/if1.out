Ident "x" = VInt 1


